lib/taste_tester/client.rb

```diff
@@ -125,14 +125,62 @@ def upload
 
     private
 
+    def tar_content(list, name)
+      full_paths = list.map do |path|
+        "#{TasteTester::Config.repo}/#{path}"
+      end
+      out = full_paths.map do |path|
+        if path.start_with?('/')
+          path = path[1..-1]
+        end
+        "--transform 's,#{path},#{name},'"
+      end
+      out.concat(
+        full_paths.each do |path|
+          "'#{path}'"
+        end,
+      )
+      # partial tar command line + list of full path used for finding
+      # symlinks
+      [out.join(' '), full_paths]
+    end
+
     def full
+      if TasteTester::Config.bundle
+        dest = "#{@server.bundle_dir}/tt.tgz"
+        excludes = []
+        cmd = [
+          [TasteTester::Config.relative_cookbook_dirs, 'cookbooks'],
```

Comment:
a name.... and it's data.... would be..... a hash.

```
cmd = {
  'cookbooks' => TasteTester::Config.relative_cookbook_dirs,
  'roles' => [TasteTester::Config.relative_role_dir],
  'databags' => [TasteTester::Config.relative_databag_dir],
}.each do |name, list|
```

Reply:
This was removed